### PR TITLE
Require PyYAML

### PR DESCRIPTION
**Pull request by chriskuehl**
Fixes https://github.com/Yelp/pgctl/issues/135

### CODE REVIEW

File: pgctl/config.py [hunk 1]
```diff
@@ -61,7 +57,7 @@ def from_file(self, filename):
                     result[key] = value
             return result
         elif filename.endswith(('.yaml', '.yml')):
-            return yaml_load(io.open(filename))
+            return yaml.safe_load(io.open(filename))
```
Comment (asottile) [inline pgctl/config.py]: this is (fwiw) the slow load -- you probably want to do the `Loader` trick to get the `CSafeLoader`

Comment (asottile) [reply pgctl/config.py]:
```python
yaml.load(
    ...,
    Loader=getattr(yaml, 'CSafeLoader', yaml.SafeLoader),
)
```

Comment (chriskuehl) [reply pgctl/config.py]: I wish this interface wasn't so bad :\

**Comment by chriskuehl** [comment]
lol guess we lost the Travis integration

File: pgctl/config.py (hunk 1 above)
Comment (asottile) [reply pgctl/config.py]: same

**Comment by chriskuehl** [comment]
close enough for now

**Pull request merged**